src/features/offer/services/useCtaWordingAndAction.ts

```diff
@@ -98,5 +116,11 @@ export const useCtaWordingAndAction = (props: {
     return
 
   const { isBeneficiary = false } = profileInfo || {}
```

Comment:
```suggestion
  const { isBeneficiary = false, bookedOffers = {} } = profileInfo || {}
```
comme ça pas besoin de rajouter de la logique en dessous